pyulog/core.py

```diff
@@ -469,23 +481,23 @@ def _read_file_definitions(self):
                 break
             header.initialize(data)
             data = self._file_handle.read(header.msg_size)
-            if header.msg_type == self.MSG_TYPE_INFO:
+            if header.msg_type_str == "INFO":
```

Comment:
Comments picked from https://github.com/PX4/pyulog/pull/41

> @bkueng: "This is now a string comparison, did you check the performance impact? I prefer an integer comparison. An enum would work too, but python has those only since 3.4."

Reply:
Not yet

Reply:
Can you revert this then, and move the `_find_sync` logic/call into `_check_file_corruption`? Then it's consistent and everything in one place.

Reply:
I did this, but it makes the _find_sync() fn a little more counter-intuitive. This is because in the `_read_file_data` function read the msg_size data bytes before verifying the msg_type, so this function has to seek back and scrape through the payload for sync_bytes.